calculator.rb

```diff
@@ -0,0 +1,91 @@
+# Calculator.rb assignment
+
+# array of arrays 
+operators = [
+  ["add", "+"], 
+  ["subtract", "-"], 
+  ["multiply", "*"], 
+  ["divide", "/"]
+]
+
+# added this operator_check long array to ensure user's input is only valid if on this list
+operators_check = 
+  ["add", "+", 
+  "subtract", "-", 
+  "multiply", "*", 
+  "divide", "/"]
+
+# only accept digits as input
+def is_number(string)
+  if string.length == 0
+    return false
+  end
+  return string.scan(/\D/).empty?
```

Comment:
This is a neat way to check if it's a number! If you found this out on the web, don't forget to cite your source!